### DISCUSSION TITLE
Drop down menu does not display for me

### DISCUSSION THREAD

**Discussion by rsmcalister** [binokochumolvarghese/lightbi-hugo#77] [Q&A] [answered]
On narrow screen/mobile device, tapping the menu button does not show dropdown menu for me. Is there a config setting for this that I am missing or something else?

Example lightbi-hugo site works.

**Comment by binokochumolvarghese** [comment]
That is coming from Bootstrap. Can you check you are loading all the files correctly?

**Reply by rsmcalister** [reply, marked as answer]
Rechecked my setup. Missed an error message related to submodules loading.

Reset things and it works fine.

Thanks for the quick response.
